.github/workflows/docker-build-test.yml

```
@@ -0,0 +1,34 @@
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Cache Docker layers
+        uses: actions/cache@v2
```

actionlint (1.7.4)

the runner of "actions/cache@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)